syllabiShare/views.py

```diff
@@ -72,7 +72,7 @@ def about(request):
 
 
 def admin(request):
-    if request.user.is_superuser:
+    if not request.user.is_superuser:
```

Comment:
This definitely seems wrong

Reply:
oof i was deleting db.sqlite too much yester and was tired of giving myself superuser lmao